src/Cassandra/Responses/Response.cs

```diff
@@ -25,12 +26,22 @@ internal class Response
         /// <summary>
         /// Big-endian binary reader of the response frame
         /// </summary>
-        protected FrameReader Reader { get; set; }
+        protected FrameReader Reader { get; }
 
         /// <summary>
         /// Identifier of the Cassandra trace 
         /// </summary>
-        protected internal Guid? TraceId { get; set; }
+        protected internal Guid? TraceId { get; }
+
+        /// <summary>
+        /// Warnings from the server which were generated by the server to go along with this response.
+        /// </summary>
+        public string[] Warnings { get; }
+
+        /// <summary>
+        /// Gets or sets the incoming custom payload.
```

Comment:
The comment says, "Gets or sets", but this is no longer publicly settable.

Reply:
+1 I'll change that